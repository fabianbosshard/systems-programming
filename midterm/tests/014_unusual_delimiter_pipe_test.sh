exec "$1"  format=@b:@a delim="|" <<EOF
Cherry|6|12
Berry|8|15
EOF
