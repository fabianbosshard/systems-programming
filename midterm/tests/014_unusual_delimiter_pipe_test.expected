6:Cherry
8:Berry
